test_scripts/API/PerformAudioPassThru/012_ATF_P_PerformAudioPassThru_WARNINGS_Valid_Request_Mandatory_Only_Without_audioPassThruIcon.lua

```diff
@@ -0,0 +1,97 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PerformAudioPassThru] SDL must transfer request to HMI in case "audioPassThruIcon" param was omited in request from mobile app
+-- [HMI API] UI.PerformAudioPassThru request/response
```

Comment:
@maria-mih, please reference non-functional requirement for TTS:
[HMI API] TTS.Speak request/response